egs/librispeech/ASR/pruned_transducer_stateless6/conformer.py

```diff
@@ -0,0 +1,1064 @@
+#!/usr/bin/env python3
```

Comment:
What are the differences between this file and `pruned_transducer_statelss2/conformer.py`?

Reply:
The middle layer (layer-5; 0-based) from student model is needed to do distillation.

Reply:
The first commit of this pr is directly copied from pruned_transducer_stateless2.
It would be easy if you specify the commits to view the modified places.